coral-visualization/src/main/java/com/linkedin/coral/vis/RelNodeVisualizationShuttle.java

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.vis;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.rel.RelShuttleImpl;
+import org.apache.calcite.rel.core.TableFunctionScan;
+import org.apache.calcite.rel.core.TableScan;
+import org.apache.calcite.rel.logical.LogicalAggregate;
+import org.apache.calcite.rel.logical.LogicalCorrelate;
+import org.apache.calcite.rel.logical.LogicalFilter;
+import org.apache.calcite.rel.logical.LogicalIntersect;
+import org.apache.calcite.rel.logical.LogicalJoin;
+import org.apache.calcite.rel.logical.LogicalMinus;
+import org.apache.calcite.rel.logical.LogicalProject;
+import org.apache.calcite.rel.logical.LogicalSort;
+import org.apache.calcite.rel.logical.LogicalUnion;
+import org.apache.calcite.rel.logical.LogicalValues;
+
+import com.linkedin.coral.hive.hive2rel.rel.HiveUncollect;
+
+import guru.nidi.graphviz.attribute.Label;
+import guru.nidi.graphviz.model.Factory;
+import guru.nidi.graphviz.model.Link;
+import guru.nidi.graphviz.model.LinkTarget;
+import guru.nidi.graphviz.model.Node;
+
+import static guru.nidi.graphviz.model.Factory.*;
+
+
+public class RelNodeVisualizationShuttle extends RelShuttleImpl {
+  private final Map<RelNode, Node> nodeMap;
+
+  public RelNodeVisualizationShuttle() {
+    this.nodeMap = new HashMap<>();
+  }
+
+  @Override
+  public RelNode visit(LogicalAggregate aggregate) {
+    super.visit(aggregate);
+    return aggregate;
+  }
+
+  @Override
+  public RelNode visit(TableScan scan) {
+    super.visit(scan);
+    nodeMap.put(scan, node(String.join(".", scan.getTable().getQualifiedName())));
+    return scan;
+  }
+
+  @Override
+  public RelNode visit(TableFunctionScan scan) {
+    super.visit(scan);
+    nodeMap.put(scan, node(scan.getCall().toString()));
+    return scan;
+  }
+
+  @Override
+  public RelNode visit(LogicalValues values) {
```

Comment:
can we drop `LogicalValues` RelNode when its rowType is `RecordType(Integer Zero)`? 
what is the value added in the viz by expressing this node?